### PR TITLE
[ADD]Hotjar Website Analytics

### PR DESCRIPTION
**Pull request by lmgr0312**
@alan196 

Regards!

### CODE REVIEW

File: website_analytics_hotjar/i18n/af.po [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Translation of Odoo Server.
```
Comment (alan196) [inline website_analytics_hotjar/i18n/af.po]: remove po files not related to this module

File: website_analytics_hotjar/models/website.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+from odoo import models, fields
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    hotjar_analytics_id = fields.Char(
+        'Hotjar Site ID', help='The ID Hotjar uses to identify the website',
+        default=1)
```
Comment (alan196) [inline website_analytics_hotjar/models/website.py]: why do you need a default?

Comment (lmgr0312) [reply website_analytics_hotjar/models/website.py]: The default value was in the module I took by reference.. IYKWIM

File: website_analytics_hotjar/i18n/af.po (hunk 1 above)
Comment (lmgr0312) [reply website_analytics_hotjar/i18n/af.po]: ok... i'm on my way

Comment (alan196) [reply website_analytics_hotjar/i18n/af.po]: ok... it's not important... only was a question jejeje 😆

File: website_analytics_hotjar/models/website.py (hunk 2 above)
Comment (alan196) [reply website_analytics_hotjar/models/website.py]: ok... it's not important... only was a question jejeje

Comment (lmgr0312) [reply website_analytics_hotjar/models/website.py]: The changes are already done.. :+1:

**Pull request merged**